### ISSUE TITLE
CH17 Vagrant Issue

### ISSUE THREAD

**Issue by ADubhlaoich** [miguelgrinberg/microblog#209]
Hello. I'm having difficulty figuring out what the precise sequence of commands is necessary to log into the Vagrant box as the `ubuntu` user. I've rebuilt/destroyed the box a number of times while trying to get this to work.

- `vagrant up`
- `vagrant ssh`
- `(vagrant) ssh ubuntu@192.168.33.10`

This attempt to log into the ubuntu user shoots out a somewhat expected `Permission denied (publickey).` error; with some cursory googling a number of solutions presented suggest deleting the default `ubuntu` user and making a new one to overcome this, or writing an entire new Vagrantfile configuration to work around `ssh` logins.

The phrasing at the top of the 'Password-less Logins' sections suggests there's no other configuration required, unless I've totally dropped the ball on the intended workflow for logging in as the 'ubuntu' user.

 In short, I have no clue if I'm meant *find* the SSH keys (if they're provided), or if I'm meant to generate them from the Vagrant box and add them to the `ubuntu` user without being able to access the account.

**Comment by miguelgrinberg** [comment]
This depends on the Vagrant box that you used. Some boxes configure a `vagrant` account, not `ubuntu`. The box that I used was the official one from Canonical.

**Comment by ADubhlaoich** [comment]
I was unaware of the nuance around Vagrant boxes - I assumed they'd act in a uniform way! Thanks for the help; in actual deployment I think I'm most likely to use Docker.

I've been enjoying the book, and meant to say such in Limerick - maybe elsewhere later this year.
